pkgs/applications/editors/emacs/default.nix

```diff
@@ -34,13 +34,19 @@ stdenv.mkDerivation rec {
     sha256 = "0cwgyiyymnx4xdg99dm2drfxcyhy2jmyf0rkr9fwj9mwwf77kwhr";
   };
 
-  patches = lib.optional stdenv.isDarwin ./at-fdcwd.patch;
+  patches = (lib.optional stdenv.isDarwin ./at-fdcwd.patch) ++ [
```

Comment:
If these patches are available from a stable URL, e.g. cgit, I think it's usually preferred to `fetchurl` them from there.